webserver/webserver/src/test/java/io/helidon/webserver/http1/WriteBufferTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.webserver.http1;
+
+import io.helidon.webserver.WebServer;
+import io.helidon.webserver.http.Handler;
+import io.helidon.webserver.http1.Http1ServerResponse.BlockingOutputStream;
+import io.helidon.webserver.http1.Http1ServerResponse.ClosingBufferedOutputStream;
+import org.junit.jupiter.api.Test;
+
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+
+import io.helidon.webserver.http.ServerResponse;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+class WriteBufferTest {
+
+    /**
+     * Test that a simple response can be sent using the {@link ServerResponse#outputStream()} using the default
+     * (non-zero) write buffer.
+     */
+    @Test
+    void defaultWriteBufferTest() throws Exception {
```

Comment:
Tests that require startup of a server should go to:
`webserver/tests/webserver`
where you can also use the `@ServerTest` annotation. See other tests in the module.
If you do not feel like changing the PR, let me know and I will take it over and do the rest.
Thanks for your contribution!

Reply:
Thanks for clarifying this. I'll gladly move the tests. How would I go about actually running the tests there? They don't seem to be part of the `webserver` module, and if I try using `webserver/tests` as a standalone project, compilation fails due to missing dependencies. I'm sure I'm missing something very obvious here.

Reply:
Never mind, I forgot to activate the `tests` profile. I moved the relevant test to the correct module.